### PR TITLE
Portfolio Project

### PR DESCRIPTION
**Pull request by azy64**
This pull request is for the portfolio project.
I did a website for desktop but not for mobile.

### CODE REVIEW

**Review by barackm — CHANGES_REQUESTED** [review]
Hi @azy64, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# STATUS: Invalid Code Review Request 🔴 

You have submitted a project, where all the linter checks are not passing, which means they are not green. Please let us fix the errors and request fr another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,11 +1,4 @@
-# Set up a "Hello Microverse" project
```
Comment (barackm) [inline README.md:1]: - [ ] Great job on adding a README to your project. but it seems to miss important information, remember the repository has to be professional. and the README is what recruiters see first, that's why we need to keep it professional. You can use [this template](https://github.com/microverseinc/readme-template) and customize it.

**Review by barackm — CHANGES_REQUESTED** [review]
Hi @azy64, 👋 

You have done an amazing job so far! 👏👏👏
However, there are some issues that you still need to work on to go to the next project. But remember you are almost there 😉  😉 !

# STATUS: Required Changes ♻️
### _Check the comments under the review._ 👇 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
### **Hi @azy64** 

### **Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**